src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    protected double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        this.color = color;
```

Comment:
The 'color' variable should be set using the superclass constructor. Replace 'this.color = color;' with 'super(color);' in the Circle constructor.

Please recheck all classes